group/iab/IAB_Decisions.md

```diff
@@ -21,45 +22,88 @@ IAB decision taking is documented in RFC 2850:
+Votes can either use open or secret ballots. The first step in the process is
+for the IAB chair to determine whether it will be held open or secret.
 
-## IAB default voting rules for appointments, confirmation of slates and other deadline driven decisions
-The default voting rules are as follows:
+- By default, votes should be open.
 
-In order to make an appointment or confirm a candidate, the IAB requires a simple majority of the non recused voting members of the IAB to vote in the affirmative.
+- Open ballots are only shared with voting members (IAB members minus
+  disqualified and recused members).
 
-When the voting will be taken in a meeting (physical, audioconference, jabber), the fact that a given appointment or confirmation will be voted in a given regular business meeting MUST be announced in the agenda for that meeting which MUST be posted 24 hrs in advance. If the voting will be performed in any irregularly scheduled meeting (i.e. meetings other than the regular IAB meeting or in a regular IAB tech chat) then, this MUST be announced a week in advance.
+- Open ballots can be submitted either by email to the list of voting members
+  or as voice votes during an IAB teleconference.
 
-The members who cannot participate in the discussion will not be able to cast their vote for this particular round of voting.
+- Secret ballots are only shared with the Executive Director and Chair.
 
-When the voting will take place via email ("e-vote"), a ballot with the question will be sent to the IAB by the IAB Chair or the IAB Executive Administrative Manager, and the e-vote will remain open until all IAB members have responded; IAB members who have not responded to the e-vote before the next IAB meeting will be asked to cast their vote during that meeting. The results of the e-vote will be recorded in the minutes of the next IAB meeting.
+- If there is no consensus among voting members on whether a vote should be
+  open or secret, the vote will be by secret ballot.
 
-By default, the voting will be non-blind.
+- If a vote starts off open but two voting members then request that it be
+  secret, then the voting restarts with secret ballots.
 
-The IAB MAY use alternative voting rules. In order to do that the IAB chair MUST announce the alternative voting rules to the IAB at least one week before the voting takes place. If a member of the IAB objects to the alternative voting rules, the resolution of the conflict will be addressed following usual RFC 2850 guidelines.
+## Normal Process
 
-&nbsp;
-&nbsp;
-&nbsp;
+The IAB chair (or executive director on the instructions of the chair) will 
+typically follow the following process:
 
----
+- If the request for a vote comes from an external source (e.g., the NomCom
+  provides its IESG slate for confirmation), then, as soon as possible after
+  receiving the request, the IAB chair adds the vote topic to the agenda of the
+  next IAB teleconference. If the next meeting is less than 24 hours away, the
```

Comment:
Should this be 24 or 48? (Based on retreat discussion on this, I don't feel strongly personally)

Reply:
Yep I just forgot to fix this after the discussion. Switched to 48 now.